velox/common/caching/SsdFile.cpp

```diff
@@ -899,12 +946,14 @@ void SsdFile::readCheckpoint(std::ifstream& state) {
     evictedMap.insert(region);
   }
   for (;;) {
-    const uint64_t fileNum = readNumber<uint64_t>(state);
+    const auto fileNum = readNumber<uint64_t>(state);
     if (fileNum == kCheckpointEndMarker) {
       break;
     }
-    const uint64_t offset = readNumber<uint64_t>(state);
-    const auto run = SsdRun(readNumber<uint64_t>(state));
+    const auto offset = readNumber<uint64_t>(state);
+    const auto fileBits = readNumber<uint64_t>(state);
+    const auto checksum = readNumber<uint32_t>(state);
```

Comment:
Do we need to handle the case that checksum is not enabled and checkpoint file has or hasn't checksum? Thanks!